distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -941,10 +942,8 @@ type Integer
 
     parse_builtin text radix = @Builtin_Method "Integer.parse"
 
-Comparable.from (that:Number) =
-    case that.is_nan of
-        True -> Incomparable
-        False -> Default_Ordered_Comparator
```

Comment:
This used to be a huge performance bottleneck.

Reply:
Is there some speedup after the rewrite?

Reply:
> Is there some speedup after the rewrite?

Yes. `sieve.enso` is 2x faster, and `EqualsBenchmarks.equalsPrimitives` is 10% faster. I have not tried other benchmarks, but I expect similar results, certainly no regressions.